.github/workflows/release.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         ATLAS_VERSION: ${{needs.build.outputs.ATLAS_VERSION}}
       run: |
         export GH_TOKEN=${{ secrets.GITHUB_TOKEN }}
-        SEMANTIC_VERSION=v${ATLAS_VERSION/#atlas-}
+        SEMANTIC_VERSION=v${ATLAS_VERSION/#atlas-cardano-}
```

Comment:
Since the package name has changed, the cardano suffix must be also removed.

If this is not done, then the release step fails later in the release workflow, since it is looking for a tag in the git repository based on the atlas version output.

This change should address this issue, by removing the whole cabal package name, and not only atlas, which was the old name, but to fix the name collision with the existing Hackage package name, we added the cardano suffix.